scripts/package

```diff
@@ -15,12 +15,18 @@ cp ../bin/pipeline ./pipeline/
 
 TAG=${TAG:-${VERSION}${SUFFIX}}
 REPO=${REPO:-rancher}
+#Tag for dependent images
+PIPELINE_UI_TAG=v0.1.1
```

Comment:
Shall we keep the tag name same as its repository? Like, in Pipeline UI, its tag is v1.0.1?

Reply:
https://github.com/rancher/pipeline-ui/releases

Reply:
I prefer using this bc we don't keep consistency with that before

Reply:
We don't take it seriously before, I think. It may cause some confusion using different tag.